akka-http-core/src/main/scala/akka/http/impl/util/package.scala

```diff
@@ -127,6 +127,53 @@ package object util {
 
 package util {
 
+  import akka.http.scaladsl.model.{ ContentType, HttpEntity }
+  import akka.stream.{ Outlet, Inlet, FlowShape }
+  import scala.concurrent.duration.FiniteDuration
+
+  private[http] class ToStrict(timeout: FiniteDuration, contentType: ContentType)
```

Comment:
Well, the DSL zoo is a bit too wide right now, in fact, we will remove FlexiMerge and FlexiRoute. We can certainly add more friendly mid-tier DSLs later, but these are always missing "one more feature" and I am happy that we have at least a fairly powerful bottom-tier DSL now. I.e. in the future yes, right now, probably no. 

But you can easily add layers in HTTP on top of the GraphStage if needed. For this stage it didn't feel worth it, but if you find nice abstractions we can always migrate them to the main library.